easybuild/easyconfigs/n/nanopolish/nanopolish-0.10.2-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,46 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+# Fred Hutchenson Cancer Research Center
+#
+#  June 13, 2017 0.7.1
+#  Dec 31, 2018, 0.10.2
+easyblock = 'MakeCp'
+
+name = 'nanopolish'
+version = '0.10.2'
+versionsuffix = '-Python-2.7.15'
+
+homepage = 'http://github.com/jts/nanopolish'
+description = """Software package for signal-level analysis of Oxford Nanopore
+ sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [('https://github.com/jts/nanopolish/archive/')]
+sources = [
+    {'download_filename': 'v%(version)s.tar.gz',
+     'filename': SOURCE_TAR_GZ}
```

Comment:
@fizwit Why is `fast5` not also included as a source file, like is done in #7376?

Note that this also requires additional `buildopts`

Reply:
I built fast5 as its own package. Not sure which way is best; sources or dependencies.

Reply:
Are you sure `nanopolish` is able to find it when you use `fast5` as a dependency?